device_doctor/lib/src/utils.dart

```diff
@@ -11,6 +11,8 @@ import 'package:process/process.dart';
 
 import 'dart:convert' show utf8;
 
+const String kDevicePropertiesFilename = '.properties';
+const String kDeviceFailedHealthcheckFilename = '.failedhealthcheck';
```

Comment:
It might be easier to handle a .healthcheck file whose contents are if it succeeded or failed, and any additional info in future, rather than a negative file, which is specific for a certain case only.

Reply:
Created a follow up PR: https://github.com/flutter/cocoon/pull/1065/files to include status and details.